packages/block-library/src/audio/edit.native.js

```diff
@@ -153,46 +154,70 @@ function AudioEdit( {
 	}
 
 	return (
-		<>
```

Comment:
I realized that the React fragment tag was removed. What is the rationale for using it or not in the context of our native components because I have seen it being used in some of them.

Reply:
In this case render was returning 2 JSX elements side by side:
```jsx
return <InspectorControls/> <MediaUpload/>;
```
And it's only allowed to return 1 JSX element from `render`. We could wrap them in a `View`, but that would render an extra view in native, so we had wrapped them in a `Fragment`, which doesn't `render` anything.

The change I made returns 1 JSX element (`TouchableWithoutFeedback`) now, so we don't need to wrap them anymore.

Reply:
Makes sense! Thanks 👍